packages/table/test/formatsTests.tsx

```diff
@@ -115,7 +119,11 @@ describe("Formats", () => {
                 The fair Ophelia! -- Nymph, in thy orisons
                 Be all my sins remembered.
             `;
-            const comp = harness.mount(<TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>);
+            const comp = harness.mount(
+                <div className={Classes.TABLE_NO_WRAP_TEXT}>
+                    <TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>
+                </div>,
```

Comment:
this is an awkward change. does `TruncatedFormat` by itself no longer work as expected? cuz that seems problematic (although it's unlikely to be used directly on its own like this).